src/main/java/com/book/controller/UserController.java

```diff
@@ -0,0 +1,47 @@
+package com.book.controller;
+
+import com.book.model.JwtModel;
+import com.book.model.UsersEntity;
+import com.book.service.MyUserDetailsService;
+import com.book.service.UserService;
+import com.book.util.JwtUtils;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping("/user")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+    private final JwtUtils jwtUtils;
+
+    @PostMapping ("/changePassword")
+    public ResponseEntity<?> changePassword(@RequestBody ObjectNode json) {
```

Comment:
using ObjectNode is not recommended. Try to define a POJO (a DTO object) instead